packages/devtools_shared/lib/src/utils/semantic_version.dart

```diff
@@ -6,7 +6,7 @@ import 'dart:math' as math;
 
 import 'compare.dart';
 
-class SemanticVersion with CompareMixin {
```

Comment:
Should this have release notes? Is this code imported by users?

Any code that was passing in an object other than a `SemanticVersion` into `compareTo` was a bug, but this could still be breaking?

Reply:
According to pub, these are the only packages that depend on devtools_shared (in alignment with what I would expect). None of those packages break as far as I can tell. Can you add a Changelog entry for this change?

Reply:
Sounds good; done!